neo/rawio/edfrawio.py

```diff
@@ -115,16 +116,18 @@ def _parse_header(self):
 
             stream_id = stream_characteristics.index((sr,))
             self.stream_idx_to_chidx.setdefault(stream_id, []).append(ch_idx)
-
-            signal_channels.append((ch_name, chan_id, sr, dtype, units, gain, offset, stream_id))
+            buffer_id = ""
+            signal_channels.append((ch_name, chan_id, sr, dtype, units, gain, offset, stream_id, buffer_id))
 
         # convert channel index lists to arrays for indexing
         self.stream_idx_to_chidx = {k: np.array(v) for k, v in self.stream_idx_to_chidx.items()}
 
         signal_channels = np.array(signal_channels, dtype=_signal_channel_dtype)
 
-        signal_streams = [(f"stream ({sr} Hz)", i) for i, sr in enumerate(stream_characteristics)]
+        signal_streams = [(f"stream ({sr} Hz)", i, "") for i, sr in enumerate(stream_characteristics)]
         signal_streams = np.array(signal_streams, dtype=_signal_stream_dtype)
+        # no buffer handling here
```

Comment:
```suggestion
        # TODO: find out buffers here.
```
This can be done, right? this is a single file. I am asking because you might know something that I don't.

Reply:
Ah, that makes sense. That's true, feel free to ignore this suggestion @samuelgarcia